apps/openchallenges/organization-service/src/test/java/org/sagebionetworks/openchallenges/organization/service/service/OrganizationServiceTest.java

```diff
@@ -1,81 +1,91 @@
-//     entityFromConstructor2.setAcronym(acronym);
-//     entityFromConstructor2.setCategories(Collections.emptyList());
-//     entityFromConstructor2.setChallengeContributions(Collections.emptyList());
-
-//   }
-
-//   @Test
-//   public void ConvertToEntity_ShouldReturnDtoProperties_WhenDtoPropertiesPassed() {
-//   }
-
-// }
+package org.sagebionetworks.openchallenges.organization.service.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+import java.util.Optional;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.sagebionetworks.openchallenges.organization.service.exception.OrganizationNotFoundException;
+import org.sagebionetworks.openchallenges.organization.service.model.dto.OrganizationDto;
+import org.sagebionetworks.openchallenges.organization.service.model.entity.OrganizationEntity;
+import org.sagebionetworks.openchallenges.organization.service.model.mapper.OrganizationMapper;
+import org.sagebionetworks.openchallenges.organization.service.model.repository.OrganizationRepository;
+
+@ExtendWith(MockitoExtension.class)
+class OrganizationServiceTest {
+
+  @Mock private OrganizationRepository organizationRepository;
+
+  @Mock private OrganizationMapper organizationMapper;
+
+  @BeforeEach
+  void setup() {
+    OrganizationService organizationService = new OrganizationService(organizationRepository);
+  }
+
+  @Test
+  void getOrganization_ShouldReturnOrganizationDto_WhenValidIdentifierPassed() {
+    // Create valid identifier
+    String identifier = "123";
+    Long orgId = 123L;
+    String orgLogin = String.valueOf(identifier);
+
+    OrganizationEntity orgEntity = new OrganizationEntity();
+    orgEntity.setId(orgId);
+
+    OrganizationDto expectedDto = new OrganizationDto();
+    expectedDto.setId(orgId);
+
+    /* Stubbing configuration to simulate a specific behavior of the organizationRepository mock object
+    to control its response and verify the behavior of the tested code that interacts with it. */
+    when(organizationRepository.findByIdOrLogin(orgId, orgLogin))
+        .thenReturn(Optional.of(orgEntity));
+
+    OrganizationService organizationService = new OrganizationService(organizationRepository);
+
+    // get the organization using the identifier
+    OrganizationDto response = organizationService.getOrganization(identifier);
+
+    /* Test that the organization is in the repo, is not null,
+    that the id can be pulled, findByIdOrLogin method is called,
+    and no interactions with the organizationMapper
+    (verifies interactions w/Mock object) */
```

Comment:
Use 80-char lines for code and comment

Reply:
I've made the changes to the characters on the comment line.